sys-fs/ncdu/ncdu-2.6-r1.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+VERIFY_SIG_OPENPGP_KEY_PATH=/usr/share/openpgp-keys/yoranheling.asc
+# Upstream states 0.12.x and 0.13.x support,
+# but eclass supports only one slot.
+ZIG_SLOT="0.13"
```

Comment:
Ah, seems like I spoke too soon -- indeed ZIG_MAX_SLOT/ZIG_MIN_SLOT or ZIG_COMPAT would be wise for us to look at then?

Reply:
For Rust, we're likely to do _MIN/_MAX as most stuff supports latest and usually just need a minimum version, but Zig is more in flux, right?

Reply:
This is intentional, if author notices after Zig X release that his package supports X and X-1, they should IMHO update slot to latest version exclusively, and ideally patch if not supported yet, so that there are less dependants on old Zig version.

This time ncdu was lucky because 0.12 and 0.13 were released in a (relatively) very short timeframe, nearly 1.5 month, and with less breaking changes. Usually they are at least 6 months apart and have enough time to break compatibility.

Reply:
OK, we'll see how it goes. Maybe document that rationale in the commit message.

Reply:
> OK, we'll see how it goes. Maybe document that rationale in the commit message.

Which one? ncdu, zig-build (guess but not sure), or doc-comment for ZIG_SLOT variable?

Reply:
I think the latter.

Reply:
8d852c7345ded897624c0e7440ea5db8eb957f5e